### ISSUE TITLE
NTR: Mucosal papules (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8007]
New term request Mucosal papules (elthat_120911194452):

**Comment by pnrobinson** [comment]
@MickeySegal
Is this intended for the oral mucosa?

**Comment by MickeySegal** [comment]
Renamed to "Mucosal papules" and often used for the mouth.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000957
- label: Oral mucosal papule
- parents: [Abnormal buccal mucosa morphology (HP:3000019)](https://hpo.jax.org/app/browse/term/HP:3000019)
- pmid: 
- def: A small (less than 5 mm in diameter), elevated, circumscribed, solid, palpable mass located in thte mucosa of the oral cavity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8007
- synonyms:
